mlflow/models/evaluation/default_evaluator.py

```diff
@@ -178,12 +209,19 @@ def _get_binary_classifier_metrics(*, y_true, y_pred, y_proba=None, labels=None,
             labels=labels,
             average="binary",
             pos_label=pos_label,
+            sample_weights=sample_weights,
         ),
     }
 
 
 def _get_multiclass_classifier_metrics(
-    *, y_true, y_pred, y_proba=None, labels=None, average="weighted"
+    *,
+    y_true,
+    y_pred,
+    y_proba=None,
+    labels=None,
+    average="weighted",
+    sample_weights,
```

Comment:
```suggestion
    sample_weights=None,
```

Is `None` missing or is this intentional?

Reply:
Fixed! Thanks!